napari_allencell_annotator/_tests/controller/images_controller_test.py

```diff
@@ -18,17 +18,39 @@ def setup_method(self):
         with mock.patch("napari_allencell_annotator.controller.images_controller.ImagesView"):
             self._controller = ImagesController(self._mock_viewer)
 
+    def test_load_from_csv_one(self):
```

Comment:
nit: instead of `one` `two` actually name what you're testing, even if that makes the function name really long